presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayDistinctFromOperator.java

```diff
@@ -44,7 +44,7 @@ public static boolean isDistinctFrom(
                     operator = IS_DISTINCT_FROM,
                     returnType = StandardTypes.BOOLEAN,
                     argumentTypes = {"E", "E"},
-                    convention = @Convention(arguments = {NULL_FLAG, NULL_FLAG}, result = FAIL_ON_NULL)) MethodHandle function,
+                    convention = @Convention(arguments = {BLOCK_POSITION, BLOCK_POSITION}, result = FAIL_ON_NULL)) MethodHandle function,
             @TypeParameter("E") Type type,
```

Comment:
These needs to be kept right now because different choices are required to have the same sequence of pre-binded arguments. This restriction is not an inherent. I'll try to remove it in a separate PR.